arm-graphrbac/1.6/swagger/graphrbac.json

```diff
@@ -1422,6 +1422,74 @@
           "operationName": "Objects_GetObjectsByObjectIdsNext"
         }
       }
+    },
+    "/{tenantID}/domains": {
```

Comment:
I don't believe so - I added this because this is needed in the fluent SDK development. If it's needed in PowerShell / CLI they can surely take advantage of this.